src/components/Navbar.js

```diff
@@ -0,0 +1,280 @@
+import { useState } from 'react';
+import { AiOutlineGlobal } from 'react-icons/ai';
+/* import { Link } from 'react-router-dom';
+import {
+  HOME_ROUTE,
+  CONTACT_ROUTE,
+  ABOUT_ROUTE,
+  SIGN_IN_ROUTE,
+  SIGN_UP_ROUTE,
+  PROJECT_ROUTE,
+  FAQ_ROUTE,
+} from '../routes'; */
+
+function Navbar() {
+  const [isShow, setIsShow] = useState(true);
+
+  function handleShowMenu() {
+    setIsShow(false);
+  }
+
+  return (
+    <header>
+      <nav className="navbar navbar-expand-lg shadow-md py-2 bg-white relative flex items-center w-full justify-between mb-5">
+        <div className="px-6 w-full flex flex-wrap items-center justify-between">
+          <div className="flex items-center">
+            <button
+              className="navbar-toggler border-0 py-3 lg:hidden leading-none text-xl bg-transparent text-gray-600 hover:text-gray-700 focus:text-gray-700 transition-shadow duration-150 ease-in-out mr-2"
+              type="button"
+              data-bs-toggle="collapse"
+              data-bs-target="#navbarSupportedContentY"
+              aria-controls="navbarSupportedContentY"
+              aria-expanded="false"
+              aria-label="Toggle navigation"
+            >
+              <svg
+                aria-hidden="true"
+                focusable="false"
+                data-prefix="fas"
+                className="w-5"
+                role="img"
+                xmlns="http://www.w3.org/2000/svg"
+                viewBox="0 0 448 512"
+              >
+                <path
+                  fill="currentColor"
+                  d="M16 132h416c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H16C7.163 60 0 67.163 0 76v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
+                />
+              </svg>
+            </button>
+          </div>
+          <div
+            className="navbar-collapse collapse grow items-center "
+            id="navbarSupportedContentY"
+          >
+            <ul className="navbar-nav mr-auto items-center lg:flex lg:flex-row">
+              <li className="nav-item">
+                <a
+                  href="#home"
+                  // to={HOME_ROUTE}
+                  className="flex items-center py-2 lg:px-2"
+                >
+                  <img
+                    src="https://user-images.githubusercontent.com/60944453/162048684-bbbe43f3-cd5b-4e93-908f-825ac7166ed1.PNG"
+                    alt="Logo"
+                    className="w-14"
+                  />
+                </a>
+              </li>
+              <li className="nav-item">
+                <a
```

Comment:
UI looks so good, but maybe you can make these a tags another component and then, map through them since it repeats itself here and they are almost same things. What do you thing?

Reply:
Thank you for your feedback. It's good to hear a different perspective. Maybe I can revise the code at the optimization stage.